### PR TITLE
Add to onboarding reproduction logs

### PR DESCRIPTION
**Pull request by salinaria**
I used Google Colab environment with T4 GPU.
Instructions are clear and runnable.
All the steps went pretty well and successful.

### CODE REVIEW

**Comment by salinaria** [comment]
I used Google Colab environment with T4 GPU.
Instructions are clear and runnable.
All the steps went pretty well and successful.

**Comment by lintool** [comment]
Please continue along the onboarding path until you reach the end of the exercises, and then we'll take a look all at once.

**Comment by salinaria** [comment]
I downloaded the dataset and I run the project on my lap top with 2GB GPU and 16GB Ram.
All the steps done successfully.

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -443,4 +443,5 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@shakibaam](https://github.com/shakibaam) on 2023-11-03 (commit [`2f3e7d5`](https://github.com/castorini/anserini/commit/2f3e7d50ef87a687f519ec0c4874807845a71078))
 + Results reproduced by [@gitHubAndyLee2020](https://github.com/gitHubAndyLee2020) on 2023-11-03 (commit [`2f3e7d5`](https://github.com/castorini/anserini/commit/2f3e7d50ef87a687f519ec0c4874807845a71078))
 + Results reproduced by [@Melissa1412](https://github.com/Melissa1412) on 2023-11-04 (commit [`cf459b3`](https://github.com/castorini/anserini/commit/cf459b39ecc2c92a400210a9ba84736bdb5d4422))
-+ Results reproduced by [@aliranjbari](https://github.com/aliranjbari) on 2023-11-06 (commit [`d2fb8a5`](https://github.com/castorini/anserini/commit/d2fb8a51f130c40aed99eee10ce34fc3f66869c7))
\ No newline at end of file
++ Results reproduced by [@aliranjbari](https://github.com/aliranjbari) on 2023-11-06 (commit [`d2fb8a5`](https://github.com/castorini/anserini/commit/d2fb8a51f130c40aed99eee10ce34fc3f66869c7))
++ Results reproduced by [@salinaria](https://github.com/salinaria) on 2023-11-09 (commit [`82634e8`](https://github.com/salinaria/anserini/commit/82634e80a98f714a88cbb1f57ed19d27d3384d90))
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:447]: you need to refer to the commit id on the main trunk of anserini, not you own clone... same below

**Review by lintool — APPROVED** [review]

**Pull request merged**